web/src/components/LocationHeader.tsx

```diff
@@ -54,16 +57,39 @@ const LocationHeader = (props: PropsType): ReactElement => {
 
   const { t } = useTranslation('layout')
 
-  const getActionItems = (): Array<ReactNode> => [
-    <HeaderActionBarItemLink key='search' href={searchPath} text={t('search')} iconSrc={searchIcon} />,
-    ...(!buildConfig().featureFlags.fixedCity
-      ? [<HeaderActionBarItemLink key='location' href={landingPath} text={t('changeLocation')} iconSrc={landingIcon} />]
-      : []),
+  const getActionItems = (viewportSmall: boolean): Array<ReactNode> => {
```

Comment:
Optional: Maybe you could refactor `getActionItems` and `getKebabItems` to not be functions as it is unnecessary, i.e. just
```ts
  const actionItems = viewportSmall ? [...] : [...]
  const kebabItems = [...]
```